packages/camera/tizen/src/camera_device.cc

```diff
@@ -614,14 +795,31 @@ bool CameraDevice::Open(std::string image_format_group) {
 
   auto value = std::make_unique<flutter::EncodableValue>(map);
   camera_method_channel_->Send(CameraEventType::kInitialized, std::move(value));
+
   return true;
 }
 
+void CameraDevice::PauseVideoRecording(
+    std::unique_ptr<flutter::MethodResult<flutter::EncodableValue>> &&result) {
+  LOG_DEBUG("enter");
+  PauseRecorder();
```

Comment:
The PauseRecorder()'s return value is a boolean type. If it returns false, should not this function also return Error()?

Reply:
You're right, I designed the return type of all wrapping methods as Boolean, but all public APIs in CameraDevice doesn't thrown any errors and I rarely checked the return value yet.
I have a plan to mimic the error handle behavior on Android after I finish implementing features roughly. 
I'd like to implement these things later If you don't mind.